test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -69,6 +79,8 @@ public HubConnectionTests()
                 {
                     //TODO: Get rid of this. This is to prevent "No channel" failures due to sends occuring before the first poll.
                     await Task.Delay(500);
+                    EnsureConnectionEstablished(connection);
```

Comment:
why do we need this? awaiting ConnectAsync should be enough to ensure the connection was started